youtube_dl/extractor/imgur.py

```diff
@@ -21,12 +21,25 @@ class ImgurIE(InfoExtractor):
             'title': 'MRW gifv is up and running without any bugs',
             'description': 'The Internet\'s visual storytelling community. Explore, share, and discuss the best visual stories the Internet has to offer.',
         },
+    }, {
+        'url': 'https://imgur.com/A61SaA1',
+        'info_dict': {
+            'id': 'A61SaA1',
+            'ext': 'mp4',
+            'title': 'MRW gifv is up and running without any bugs',
+            'description': 'The Internet\'s visual storytelling community. Explore, share, and discuss the best visual stories the Internet has to offer.',
+        },
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
 
+        sources = re.findall(r'<source src="([^"]+)" type="([^"]+)"', webpage)
```

Comment:
I don't really have a strong preference on the failure mode; it's whatever you want.

If I remove this block, you get this:

```
Traceback (most recent call last):
  File "youtube_dl/YoutubeDL.py", line 643, in extract_info
    ie_result = ie.extract(url)
  File "youtube_dl/extractor/common.py", line 269, in extract
    return self._real_extract(url)
  File "youtube_dl/extractor/imgur.py", line 48, in _real_extract
    webpage, 'video elements')
  File "youtube_dl/extractor/common.py", line 531, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
```

If I replace it with this:

```
        formats = []
        video_elements = str_or_none(self._search_regex(
            r'(?s)<div class="video-elements">(.*?)</div>',
            webpage, 'video elements', fatal=False))
        if not video_elements:
            raise ExtractorError(
                'No sources found for video %s' % video_id, expected=True)
```

Then you get:
```
python -m youtube_dl https://imgur.com/hL8R5YG
[Imgur] hL8R5YG: Downloading webpage
WARNING: unable to extract width; please report this issue on http://yt-dl.org/bug
WARNING: unable to extract height; please report this issue on http://yt-dl.org/bug
WARNING: unable to extract video elements; please report this issue on http://yt-dl.org/bug
ERROR: No sources found for video hL8R5YG
```